README.md

```diff
@@ -564,6 +564,60 @@ public class AWSSecretsManagerPluginSample {
 }
 ```
 
+## Read-Write Splitting Plugin
+
+The read-write splitting plugin adds functionality to switch between writer/reader instances via calls to the `JdbcConnection#setReadOnly` method. Upon calling `setReadOnly(true)`, the plugin will establish a connection to a random reader instance and direct subsequent queries to this instance. Future calls to `setReadOnly` will switch between the established writer and reader connections according to the boolean argument you supply to the `setReadOnly` method.
+
+### Loading the Read-Write Splitting Plugin
+
+The read-write splitting plugin is not loaded by default. To load the plugin, set the `connectionPluginFactories` connection parameter:
+
+```
+final Properties properties = new Properties();
+properties.setProperty("connectionPluginFactories", ReadWriteSplittingPluginFactory.class.getName());
+```
+
+If you would like to load the read-write splitting plugin alongside the failover and enhanced failure monitoring plugins, the read-write splitting plugin must be the first plugin in the connection chain, otherwise failover exceptions will not be properly processed by the plugin:
+
+```
+final Properties properties = new Properties();
+properties.setProperty(
+        "connectionPluginFactories",
+        String.format("%s,%s,%s",
+                ReadWriteSplittingPluginFactory.class.getName(),
+                FailoverConnectionPluginFactory.class.getName(),
+                NodeMonitoringConnectionPluginFactory.class.getName())
+```
+
+### Reader load-balancing
+
+The plugin can also load-balance queries among available reader instances by enabling the `loadBalanceReadOnlyTraffic` connection parameter. This parameter is disabled by default. To enable it, set the following connection parameter:
+
+```
+final Properties properties = new Properties();
+properties.setProperty("loadBalanceReadOnlyTraffic", "true");
+```
+
+Once this parameter is enabled, load-balancing will automatically be performed for reader instances when the connection has been set to read-only mode via `JdbcConnection#setReadOnly`. Load-balancing is performed by randomly selecting new reader instances at transaction boundaries.
```

Comment:
These sentences were a bit tough to read; I hope this suggestion is still technically accurate:

Once this parameter is enabled, you can enable load-balancing for reader instances by setting the `JdbcConnection#setReadOnly` property to read-only mode. Load-balancing will distribute queries across randomly selected new reader instances at transaction boundaries.

Reply:
Thanks Susan, yes that is still accurate, I'll make that change

Reply:
Ended up going with slightly different wording because I thought it was more clear/accurate. Let me know if you would like me to change it:

Once this parameter is enabled, queries will be load-balanced among reader instances after calling `setReadOnly(true)` on the Connection object. Load-balancing will switch to a new randomly selected reader instance at each transaction boundary.

Reply:
Sounds good, thanks!